### PR TITLE
#331 api edit profile request

### PR DESCRIPTION
**Pull request by elli95**
You can now edit avatar, first/last name, title, skills and about for the student profiles.
Added an extra input field and divided first and last name between them.

### CODE REVIEW

File: src/js/api/users/editStudent.js [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
 export async function editStudent(profile) {
-  const { id } = profile;
-  const profileURL = apiPath + action + `${id}`;
+  // const { id } = profile;
+  const id = localStorage.getItem('id');
+  const newId = id.replace(/^"|"$/g, '');
+
+  const profileURL = apiPath + action + newId;
+
+  console.log('hi', profile);
```
Comment (Olekrr) [inline src/js/api/users/editStudent.js:22]: We should generally avoid having console.logs in our final product, as frequent logging can impact performance or inadvertently expose sensitive information

File: src/js/api/users/editStudent.js [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
+  const accessToken = getToken('token');
+  const newAccessToken = accessToken.replace(/^"|"$/g, '');
+  console.log('accessToken', newAccessToken);
```
Comment (Olekrr) [inline src/js/api/users/editStudent.js:25]: Especially when it comes to accesstokens

**Review by Olekrr — CHANGES_REQUESTED** [review]
Looks mostly good to me, but the console logs needs to be removed.

Well done!

File: src/js/api/users/editStudent.js (hunk 1 above)
Comment (elli95) [reply src/js/api/users/editStudent.js:22]: Got it.

File: src/js/api/users/editStudent.js (hunk 2 above)
Comment (elli95) [reply src/js/api/users/editStudent.js:25]: Sorry, forgot about it, will keep an eye out for it going forward.

**Review by Olekrr — APPROVED** [review]

**Comment by Olekrr** [comment]
Changes approved, merging this

**Pull request merged**